guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: guide
+title: Units & Symbols
+nav_order: 1
+parent: Payments
+permalink: /guide/payments/units-and-symbols/
+typora-root-url: ./images
+---
+
+# Units and symbols
+
+<!--Don't forget to update date -->
+11 December 2020
+
+Bitcoin, bits , sats. The format and presentation of bitcoin values are probably amongst the most challenging for new users. The objective of this section is to provide observations of its current usage and recommendations for the best option if multiple formats exist. The goal is not to re-invent the format but to observe the current usage, and eventually to codify it through an organic approach. The surest way to achieve wide-scale adoption is to standardize what is already being used today. 
```

Comment:
1. We spoke about how these units and symbols come about and how they get adopted I think one sentence on this would be good to reinforce that these "standards" are something that comes from the community and users.

Reply:
I added a small note about millisatoshi to the page. How about we wait with more Lightning info for a future PR?